telemetry/definitions/commonDefinitions.json

```diff
@@ -550,6 +550,36 @@
             "description": "The type of the user trigger to send request to CodeWhisperer service",
             "allowedValues": ["OnDemand", "AutoTrigger"]
         },
+        {
+            "name": "codewhispererFirstCompletionLatency",
+            "type": "double",
+            "description": "The latency from user performs the triggering CW action to plugin call GenerateCompletions API for the first recommendation"
+        },
+        {
+            "name": "codewhispererEndToEndLatency",
+            "type": "double",
+            "description": "The latency from plugin start invocation to plugin call GenerateCompletions API to first completion shown in the IDE"
+        },
+        {
+            "name": "codewhispererAllCompletionsLatency",
+            "type": "double",
+            "description": "The latency from plugin makes a first call to GenerateCompletions API to receiving the response for the last call to the GenerateCompletions API."
+        },
+        {
+            "name": "codewhispererPostprocessingLatency",
+            "type": "double",
+            "description": "The latency from plugin gets Completions object to completion shown in the IDE for the first completion."
```

Comment:
The time it takes for the first completions to be displayed in the IDE after the plugin receives the initial `Completions` object.